build.yaml

```diff
@@ -0,0 +1,15 @@
+targets:
+  $default:
+    builders:
+      build_vm_compilers|entrypoint:
+        generate_for:
+          include:
+            - "test/**.vm_test.dart"
+      build_web_compilers|entrypoint:
+        generate_for:
+          include:
+            - "test/**.browser_test.dart"
+            - "example/**.dart"
```

Comment:
We should be able to get rid of this `generate_for` config altogether. The defaults work fine for this package